app/helpers/application_helper.rb

```diff
@@ -7,4 +7,20 @@ def bootstrap_class_for(flash_type)
       notice: "alert-info"
     }.stringify_keys[flash_type.to_s] || flash_type.to_s
   end
+  
+  def avatar_path(object, options = {})
+    size = options[:size] || 180
+    default_image = options[:default] || "mp"
+    base_url =  "https://secure.gravatar.com/avatar"
+    base_url_params = "?s=#{size}&d=#{default_image}"
+    
+    if object.respond_to?(:avatar) && object.avatar.attached? && object.avatar.variable?
+      object.avatar.variant(resize_to_fill: [size, size, { gravity: 'Center' }])
+    elsif object.respond_to?(:email) && object.email
+      gravatar_id = Digest::MD5::hexdigest(object.email.downcase)
+      "#{base_url}/#{gravatar_id}#{base_url_params}"
+    else
+      "#{base_url}/00000000000000000000000000000000#{base_url_params}"
```

Comment:
Just curious, what's this do? Is this the default image?

Reply:
Hi Chris! Thanks for taking the time to review this PR. Yes, `"#{base_url}/00000000000000000000000000000000#{base_url_params}"` returns the [default image](https://en.gravatar.com/site/implement/images/).